src/client/components/AddMeal.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react"
+
+export default function AddMeal() {
+    const [title, setTitle] = useState("")
+    const [description, setDescription] = useState("")
+    const [location, setLocation] = useState("")
+    const [max_reservations, setMax_reservations] = useState()
+    const [price, setPrice] = useState()
+    const handleTitle = (event) => {
+        setTitle(event.target.value)
+    }
+    const handleDescription = (event) => {
+        setDescription(event.target.value)
+    }
+    const handleLocation = (event) => {
+        setLocation(event.target.value)
+    }
+    const handleMax_reservations = (event) => {
+        setMax_reservations(event.target.value)
```

Comment:
```suggestion
        setMaxReservations(event.target.value)
```

Consistent naming ☝🏼